src/sql/workbench/api/browser/mainThreadModelViewDialog.ts

```diff
@@ -213,7 +214,7 @@ export class MainThreadModelViewDialog implements MainThreadModelViewDialogShape
 
 	public $openWizard(handle: number): Thenable<void> {
 		let wizard = this.getWizard(handle);
-		this._dialogService.showWizard(wizard);
+		this._dialogService.showWizard(wizard, { hasBackButton: false, width: wizard.width, hasErrors: true });
```

Comment:
Where these the defaults before for these other options?

Reply:
i have updated the code to reflect the source.